acertmgr.py

```diff
@@ -205,10 +205,12 @@ def complete_config(domainconfig, globalconfig):
                 import json
 
                 config = json.load(config_fd)
-            except json.JSONDecodeError:
```

Comment:
Traceback (most recent call last):
  File "./acertmgr.py", line 208, in <module>
    except json.JSONDecodeError:
AttributeError: 'module' object has no attribute 'JSONDecodeError'